test/test-parse.cpp

```diff
@@ -320,8 +320,9 @@ TEST_F(Test_Parse, utter_garbage) {
     assert_diagnostics(
```

Comment:
Not your patch: I probably should have documented why I introduced this test in the first place. 😅 It's probably just testing a crash, so changing the diagnostics sounds fine to me.

Reply:
It's related to issue #88 based on the Git log, and it is testing a crash.